service/Dockerfile.rocky9

```diff
@@ -0,0 +1,61 @@
+FROM rockylinux:9 AS builder
```

Comment:
For services I normally base them of Alpine Python images, which results in smaller images and you don't need to build python, see: https://github.com/ynput/ayon-shotgrid/blob/develop/services/processor/Dockerfile